File: MosPosudit.Model/Requests/Payment/PayPalCreateOrderRequest.cs
using System.ComponentModel.DataAnnotations;
using MosPosudit.Model.Requests.Order;

namespace MosPosudit.Model.Requests.Payment
{
    public class PayPalCreateOrderRequest
    {
        [Required(ErrorMessage = "Order data is required")]
        public OrderInsertRequest OrderData { get; set; } = null!;
    }
}

File: MosPosudit.UI/shared/lib/dtos/payment/paypal_create_order_request.dart
import '../order/order_insert_request.dart';

class PayPalCreateOrderRequest {
  final OrderInsertRequest orderData;

  PayPalCreateOrderRequest({
    required this.orderData,
  });

  Map<String, dynamic> toJson() => {
        'orderData': orderData.toJson(),
      };
}

